Web/HTML/html-structure/elements.md

```diff
@@ -11,11 +11,38 @@ category: must-know
 
 tags:
   - introduction
-
+---
+# Elements
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
+HTML elements consist of a start and end tag with content in-between.
+Some elements:
+- Paragraph:`<p></p>`
+- Anchors: `<a></a>`
+- Headers: `<h3></h3>`
+- Path `href=""`
+
+Example:
+```
+<h1> Text in a header element </h1>
+```
+
+It is also very common for HTML elements to be nested. This is common even for the most simple websites.
+
+Example:
+```
+<p>Check out this 
```

Comment:
prefer this style (reduces amount of space, helps with readability) 
```
<p>
  Check out this 
    <a href="enki.com">
      website
    </a>
</p>
```

Reply:
Ok, noted.